app/models/maintenance_tasks/runner.rb

```diff
@@ -37,14 +37,17 @@ def initialize(run)
     #   for the Task to iterate over when running, in the form of an attachable
     #   (see https://edgeapi.rubyonrails.org/classes/ActiveStorage/Attached/One.html#method-i-attach).
     #   Value is nil if the Task does not use CSV iteration.
+    # @param arguments [Hash] the arguments to persist to the Run and to make
+    #   accessible to the Task.
     #
     # @return [Task] the Task that was run.
     #
     # @raise [EnqueuingError] if an error occurs while enqueuing the Run.
     # @raise [ActiveRecord::RecordInvalid] if validation errors occur while
     #   creating the Run.
-    def run(name:, csv_file: nil)
-      run = Run.active.find_by(task_name: name) || Run.new(task_name: name)
+    def run(name:, csv_file: nil, arguments: {})
```

Comment:
It's starting to be more and more weird that we use `run` for starting or resuming a task. We previously assumed you resume without a CSV (because otherwise we would override the CSV file), and now we ignore the arguments if you resume, which is a different behavior. Nothing to do for this PR though, just commenting.

Reply:
Yeah, I'll open a ticket to explore this, I think there were other things that came up that were making us think about splitting these up?